viite-UI/src/model/SelectedNodesAndJunctions.js

```diff
@@ -1,6 +1,7 @@
 (function (root) {
   root.SelectedNodesAndJunctions = function (nodeCollection) {
     var current = {};
+    var junctionNumbersVerified = true;
```

Comment:
Should this be false initially?

Reply:
Hmm, didn't thought about it. I would say that every nodes in the database should have their junctions with valid junction numbers. So, if we open a node its junctions should be 'verified'.

I actually don't like that local variable there, but it was the only way i found out to prevent the for each to run every time I need to check if junctions are valid or not.

Reply:
ok